spec/support/vcr_cassettes/mobile/lighthouse_letters/download_with_options.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
```

Comment:
This needed to change because I started passing options through to lighthouse in the controller.